mulog-core/src/com/brunobonacci/mulog/publisher.clj

```diff
@@ -240,6 +240,11 @@
     "com.brunobonacci.mulog.publishers.console-json/json-console-publisher"
     config))
 
+(defmethod publisher-factory :advanced-console
+  [config]
+  (load-dynamic-publisher
+   "com.brunobonacci.mulog.publishers.advanced-console/advanced-console-publisher"
+   config))
```

Comment:
Since the _advanced console publisher_ now has more publishers than just the ANSI coloring one,
I think it would be best to call this one `ansi-console-publisher` and thus `:type :ansi-console`